libs/constrained-generators/src/Constrained/Base.hs

```diff
@@ -539,6 +539,7 @@ instance HasSpec fn a => Semigroup (Spec fn a) where
   ErrorSpec e <> ErrorSpec e' = ErrorSpec (e ++ "" : (replicate 20 '-') : "" : e')
   ErrorSpec e <> _ = ErrorSpec e
   _ <> ErrorSpec e = ErrorSpec e
+  -- TODO: these can interact poorly with unsafeExists!
   MemberSpec [a] <> spec
     | a `conformsToSpec` spec = MemberSpec [a]
     | otherwise = ErrorSpec [show a, "does not conform to", show spec]
```

Comment:
What about adding another guard like
```
memberSpec [a] <>spec
  | not (suspenedSpec spec)
  | a `conformsToSpec` spec = MemberSpec [a]
  | otherwise = ErrorSpec [show a, "does not conform to", show spec]
```
Does that get rid of the poor interaction?

Reply:
yeah, that's a band-aide for now. We can probably refine it later.

Reply:
Problem is that if you have e.g. `Cartesian (SuspendedSpec ...) TrueSpec` that doesn't help

Reply:
I'll comment the cases out for now and we'll think it through a bit more later.